common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "cloth_config.voicechat.config.mute_on_join": "Mute on join",
-  "cloth_config.voicechat.config.mute_on_join.description": "If enabled, you will be automatically muted when joining a world",
+  "cloth_config.voicechat.config.mute_on_join.description": "When enabled, you will be automatically muted when joining a world.",
   "cloth_config.voicechat.config.audio_packet_threshold": "Audio packet threshold",
-  "cloth_config.voicechat.config.audio_packet_threshold.description": "The maximum amount of audio packets that are held back if a packet arrives out of order or gets dropped\nThis prevents discarding audio packets that are slightly out of order\nSet this to 0 to disable",
-  "cloth_config.voicechat.config.voice_deactivation_delay": "Deactivation delay",
-  "cloth_config.voicechat.config.voice_deactivation_delay.description": "The time it takes for the microphone to deactivate when using voice activation\nA value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, ...",
+  "cloth_config.voicechat.config.audio_packet_threshold.description": "The maximum number of audio packets that should be held back if a packet arrives out of order or is dropped.\nThis prevents audio packets that are only slightly out of order from being discarded.\nSet this to 0 to disable.",
+  "cloth_config.voicechat.config.voice_deactivation_delay": "Microphone deactivation delay",
+  "cloth_config.voicechat.config.voice_deactivation_delay.description": "The time it takes for the microphone to deactivate when using voice activation.\nA value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, and so on.",
   "cloth_config.voicechat.config.output_buffer_size": "Output buffer size",
-  "cloth_config.voicechat.config.output_buffer_size.description": "The size of the audio output buffer in packets\nHigher values mean a higher latency, but less crackles\nIncrease this value if you have an unstable internet connection",
-  "cloth_config.voicechat.config.hud_icon_scale": "HUD icon scale",
-  "cloth_config.voicechat.config.hud_icon_scale.description": "The scale of the HUD icons",
-  "cloth_config.voicechat.config.hud_icon_pos_x": "HUD icon X pos",
-  "cloth_config.voicechat.config.hud_icon_pos_x.description": "The X position of the HUD icons\nNegative values mean anchoring to the right",
-  "cloth_config.voicechat.config.hud_icon_pos_y": "HUD icon Y pos",
-  "cloth_config.voicechat.config.hud_icon_pos_y.description": "The Y position of the HUD icons\nNegative values mean anchoring to the bottom",
-  "cloth_config.voicechat.config.group_player_icon_orientation": "Group HUD icon orientation",
-  "cloth_config.voicechat.config.group_player_icon_orientation.description": "The orientation of the player icons in the group HUD",
+  "cloth_config.voicechat.config.output_buffer_size.description": "The size of the audio output buffer (in packets).\nHigher values mean a higher latency but less crackling.\nIncrease this value if you have an unstable Internet connection.",
```

Comment:
why is internet capitalized?

Reply:
I always thought there was only one correct form of this word — "Internet" with a capital letter 'I' — because it is a proper noun. However, it seems that there is an ongoing debate about this, and opinions are divided: https://en.wikipedia.org/wiki/Capitalization_of_Internet

What do you think?

Reply:
Fair enough, its fine for me either way :)

Reply:
If you prefer, I can restore the previous form. I don't have any personal preference here. For years, I just thought that "Internet" with a capital letter "I" was the only correct form, but it turns out I was wrong.

Reply:
Yeah i guess then leave it as it was before

Reply:
The previous form has already been restored by me. Enjoy!